README.rst

```diff
@@ -152,6 +121,29 @@ the following bash script:
 
     python main.py --port <PORT> --shell <Path to the terminal backend to execute>
 
+Build from source code
+-------------------------
+
+Install all the dependencies given in our `requirements file <https://github.com/spyder-ide/spyder-terminal/tree/master/requirements>`_
+depending on your OS, a distribution of `node <https://nodejs.org/>`_ and
+`yarn <https://yarnpkg.com/lang/en/>`_. Then, use the following bash script 
```

Comment:
I'd like this process to be more self-contained. I mean, we can use npm to install yarn, right? But after that, how can use that version instead of global version of yarn?

Reply:
I don't understand your question, npm and yarn are different package managers. In order to compile the project we need both of them to be installed. As it happens with some packages in anaconda, it is better for the user to install these two managers from their official pages using the official instructions.

Reply:
I understand that npm and yarn are different package managers, but you can install yarn with npm by simply running

    npm install yarn

So in order to avoid installing npm and yarn separately, I'd like to install `nodejs` from conda-forge (which also install npm), then install yarn with npm, and use both package managers from my conda environment.

That will simplify a lot the process of releasing new versions of `spyder-terminal`. So I'd like to change the release instructions a bit in order to use that workflow. Is it possible?

Reply:
Agreed with this proposal!

Reply:
I'm against this one, yarn maintainers are against it https://github.com/yarnpkg/yarn/issues/1082#issuecomment-254026649